torch/csrc/jit/codegen/cuda/graph_fuser.cpp

```diff
@@ -2042,6 +2042,9 @@ void replaceAliasOpsWithCopy(std::shared_ptr<Graph>& graph, Block* block) {
         graph->insertNode(graph->create(op_mapping[n->kind()], n->inputs(), 1));
     op_copy->output()->setType(n->output(0)->type());
 
+    // adding newly created value into alias_db;
+    alias_db->createValue(op_copy->output());
```

Comment:
Is this related to this PR?

Reply:
No, looks like I picked up an extra commit somehow, must have been out of sync.